test/JDBC/input/smalldatetimefromparts.sql

```diff
@@ -171,3 +171,22 @@ GO
 
 SELECT SMALLDATETIMEFROMPARTS (2079, 1,2,3,4) AS Result
 GO
+
+SELECT SMALLDATETIMEFROMPARTS( 2079, 1, 7, 0, 0 ) AS Result
+GO
+
+SELECT SMALLDATETIMEFROMPARTS( 2079, 3, 7, 0, 0 ) AS Result
+GO
+
+SELECT SMALLDATETIMEFROMPARTS( 1899, 1, 1, 0, 0 ) AS Result
+GO
+
+SELECT SMALLDATETIMEFROMPARTS( 2079, 7, 6, 0, 0 ) AS Result
```

Comment:
add test cases for (month, day) -> (6,6) (5,7) (6,5) (6,7) (7,1)

Reply:
Added